src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/controller/AppointmentsController.kt

```diff
@@ -83,4 +84,17 @@ class AppointmentsController(
     val actionPlanAppointment = appointmentsService.getAppointment(actionPlanId, sessionNumber)
     return ActionPlanAppointmentDTO.from(actionPlanAppointment)
   }
+
+  @PostMapping("/action-plan/{id}/appointment/{sessionNumber}/record-attendance")
```

Comment:
I'm writing the contracts, and just wondering if this should be the same as the `PATCH` endpoint that we hit to change session duration (eg /action-plan/345059d4-1697-467b-8914-fedec9957279/appointment/2) if we're updating the same object?